gensim/models/__init__.py

```diff
@@ -1,6 +1,5 @@
 """
-This package contains algorithms for extracting document representations from their raw
-bag-of-word counts.
+Extraction of documents representations from their raw bag-of-word counts.
```

Comment:
Maybe use `"""sometext."""` everywhere in headers `if len(line) <= 120` ?

Reply:
Ok